src/components/badge/VsBadge.vue

```diff
@@ -16,9 +16,9 @@ import { BBadge } from 'bootstrap-vue-next';
 
 <style lang="scss">
 .vs-badge {
-    border: 1px solid $vs-color-new-border-secondary;
+    border: 1px solid $vs-color-border-secondary;
     border-radius: $border_radius_default;
```

Comment:
Nothing to do with this PR, but what's with the underscores in the border-radius token?! does SASS not care about that? It seems to work regardless.

Reply:
Yeah thats interesting, it should be using dashes to reference the SASS variable - not sure how its working :D